packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -55,8 +53,8 @@ export interface PluginPackage {
  * This interface describes a package.json contribution section object.
  */
 export interface PluginPackageContribution {
-    configuration?: PreferenceSchema;
-    configurationDefaults?: PreferenceSchemaProperties;
+    configuration?: Object;
```

Comment:
I'm unsure the fix is to loose the Type ?

Reply:
in this case, the configuration in `package.json` is bogus, one has to validate that it confirms to `PreferenceSchema` before applying, the correct type would be something like `RecursivePartial<PreferenceSchema>`

Reply:
@benoitf fixed